docs/conf.py

```diff
@@ -27,12 +27,16 @@
 sourcedir = sys.argv[-2]
 builddir = sys.argv[-1]
 
+# Only build API docs if the user specifically requests it. On RTD we build it
+# all the time
+use_breathe = tags.has("use_breathe") or readTheDocs
+
 # Use this to turn Doxygen on or off
-useDoxygen=False
+useDoxygen=True
```

Comment:
This is set to `False` in the `dev` branch, have you confirmed that this doesn't lead to timeouts/errors with the Fortran sources?

Reply:
Yes. I was able to generate the documentation here http://ganesh-openfast.readthedocs.io/en/latest/source/dev/cppapi/index.html. The Doxyfile.rtd that is run on readthedocs only runs Doxygen on C++ files for now. My thinking was that when breathe works for Fortran and/or a different documentation strategy is adopted for generating automatic documentation for Fortran sources, the Doxyfile.rtd can be modified appropriately.